rclrs/src/qos.rs

```diff
@@ -95,10 +135,10 @@ impl From<QoSProfile> for rmw_qos_profile_t {
 impl From<QoSHistoryPolicy> for rmw_qos_history_policy_t {
     fn from(policy: QoSHistoryPolicy) -> Self {
         match policy {
-            QoSHistoryPolicy::SystemDefault => {
+            QoSHistoryPolicy::SystemDefault { .. } => {
```

Comment:
Just for my own edification, does the destructuring that you added here change anything? Since this isn't actually placing any constraints on the pattern matching or extracting any information from them, my understanding is this changed version is equivalent to the original version, is that right?

Reply:
Yep, the braces are just there because Rust expects them.